### ISSUE TITLE
fee estimates in coin control often lower than actual fees 

### ISSUE THREAD

**Issue by mdpfeiffer** [PaycoinFoundation/paycoin#344]
In Coin Control, the estimates for transfer fees are frequently lower than the actual transfer fees for transactions involving multiple inputs.

**Comment by IngCr3at1on** [comment]
Yeah being an "estimate" that doesn't surprise me. The proper fee is still displayed in the fee confirmation dialogue.

**Comment by mitchellcash** [comment]
As stated it is just an estimate so I'm not too worried about this and the fee confirmation dialogue will give you the correct fee prior to sending.

**Comment by IngCr3at1on** [comment]
Closing this per reasons mentioned above.

**Comment by mdpfeiffer** [comment]
I get that it's an estimate. But it doesn't say that it's an estimate. Would a different label help?

**Comment by IngCr3at1on** [comment]
@mdpfeiffer `approximate fee` and `estimated fee` both seem rather long, thoughts?

**Comment by mitchellcash** [comment]
Agreed. `Approximate Fee` and `Estimated Fee` I think would look to long in the Coin Control.

I would propose using `~`, like:

`Fee: ~0.001000 XPY`

**Comment by broyhill** [comment]
There is no warning that your transaction will fail to send when the fee is larger than the estimate, and you are sending the full amount of all inputs.

**Comment by IngCr3at1on** [comment]
@broyhill adjusting the transaction creates a loop in the create cycle that I'm not sure I'm comfortable with and the error that you get is clear enough to me that I personally don't think a warning is necessary (others may disagree).

The same problem of looping occurs when trying to calculate the proper fee because it can't be calculated until creating the transaction which would basically mean creating the transaction and throwing it out; it's a waste of resources imho.

I like that idea @mitchellcash

**Comment by broyhill** [comment]
The issue is, what is the exact fee? I don't want a warning, I want it to say "Sorry, the required fee is X." The error is just not clear enough.

I can say I have used the wallet a lot, and sometimes I still have to guess, and end up with .001, .002 amounts left in an input.
